### PR TITLE
Turn off certificate checking for Mavericks, as the curl is borked.

### PR DESCRIPTION
**Pull request by brendandburns**
ugh, why can't life be easier...

@zmerlynn @roberthbailey

### CODE REVIEW

**Comment by zmerlynn** [comment]
LGTM

File: cluster/gce/util.sh [hunk 1]
```diff
@@ -665,8 +665,18 @@ function kube-up {
   echo "  up."
   echo
 
+  # curl in mavericks is borked.
+  secure=""
+  if which sw_vers > /dev/null; then
+    if [[ $(sw_vers | grep ProductVersion | awk '{print $2}') = "10.9."* ]]; then
```
Comment (roberthbailey) [inline cluster/gce/util.sh]: `sw_vers -productVersion` gives you the number directly without needing grep/awk. At least it does on Yosemite. Maybe Mavericks isn't advanced enough.... :)

**Comment by roberthbailey** [comment]
@stephenR FYI

**Comment by brendandburns** [comment]
This works, it's needed for the 0.18.0 release.  Can we merge as is?  I'll adjust to use ```-productVersion``` later.

**Comment by roberthbailey** [comment]
Yes. LGTM.

@thockin Can you merge (as oncall) please?

**Pull request merged**